<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="34" ignored="0" total="37" passed="3">
  <reporter-output>
  </reporter-output>
  <suite name="Default Suite" duration-ms="129773" started-at="2020-05-18T11:36:11Z" finished-at="2020-05-18T11:38:21Z">
    <groups>
    </groups>
    <test name="QuotaAppTest" duration-ms="129773" started-at="2020-05-18T11:36:11Z" finished-at="2020-05-18T11:38:21Z">
      <class name="com.g2academy.testcases.loginmenu.TC_ForgotPassword">
        <test-method status="PASS" signature="setup()[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="setup" is-config="true" duration-ms="472" started-at="2020-05-18T11:36:11Z" finished-at="2020-05-18T11:36:11Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="PASS" signature="beforeClass()[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="beforeClass" is-config="true" duration-ms="10919" started-at="2020-05-18T11:36:11Z" finished-at="2020-05-18T11:36:22Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method status="PASS" signature="testForgotPassword(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="testForgotPassword" duration-ms="2740" started-at="2020-05-18T11:36:31Z" data-provider="dataForgotPassword" finished-at="2020-05-18T11:36:34Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[negative case - invalid phone number and email]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[not a number]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[testforgotpassword.gmail.com]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[500]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[Internal Server Error]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="7">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="8">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="9">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="10">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="11">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="12">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="13">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="14">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testForgotPassword -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="afterMethod" is-config="true" duration-ms="1" started-at="2020-05-18T11:36:34Z" finished-at="2020-05-18T11:36:34Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="FAIL" signature="testForgotPassword(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="testForgotPassword" duration-ms="2863" started-at="2020-05-18T11:36:34Z" data-provider="dataForgotPassword" finished-at="2020-05-18T11:36:37Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[negative case - invalid phone number]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[not a number]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[testforgotpasswordbackend@gmail.com]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[400]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[Phone number not registered]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="7">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="8">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="9">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="10">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="11">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="12">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="13">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="14">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [400] but found [500]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [400] but found [500]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at org.testng.Assert.assertEquals(Assert.java:662)
at com.g2academy.base.Assertion.statusCode(Assertion.java:8)
at com.g2academy.testcases.loginmenu.TC_ForgotPassword.testForgotPassword(TC_ForgotPassword.java:95)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:54)
at org.testng.internal.InvokeMethodRunnable.run(InvokeMethodRunnable.java:44)
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testForgotPassword -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="afterMethod" is-config="true" duration-ms="0" started-at="2020-05-18T11:36:37Z" finished-at="2020-05-18T11:36:37Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="PASS" signature="testForgotPassword(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="testForgotPassword" duration-ms="2488" started-at="2020-05-18T11:36:37Z" data-provider="dataForgotPassword" finished-at="2020-05-18T11:36:39Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[negative case - invalid email]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[+6281252930398]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[testforgotpassword.gmail.com]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[500]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[Internal Server Error]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="7">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="8">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="9">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="10">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="11">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="12">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="13">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="14">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testForgotPassword -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="afterMethod" is-config="true" duration-ms="0" started-at="2020-05-18T11:36:39Z" finished-at="2020-05-18T11:36:39Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="FAIL" signature="testForgotPassword(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="testForgotPassword" duration-ms="2972" started-at="2020-05-18T11:36:39Z" data-provider="dataForgotPassword" finished-at="2020-05-18T11:36:42Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[positif case - valid phone number and email]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[+6281252930398]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[testforgotpasswordbackend@gmail.com]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[OTP]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[TRUE]]>
              </value>
            </param>
            <param index="7">
              <value>
                <![CDATA[true]]>
              </value>
            </param>
            <param index="8">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="9">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="10">
              <value>
                <![CDATA[Your token has been reset]]>
              </value>
            </param>
            <param index="11">
              <value>
                <![CDATA[Password12345@@]]>
              </value>
            </param>
            <param index="12">
              <value>
                <![CDATA[Password12345@@]]>
              </value>
            </param>
            <param index="13">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="14">
              <value>
                <![CDATA[+6281252930398]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [200] but found [500]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [200] but found [500]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at org.testng.Assert.assertEquals(Assert.java:662)
at com.g2academy.base.Assertion.statusCode(Assertion.java:8)
at com.g2academy.testcases.loginmenu.TC_ForgotPassword.testForgotPassword(TC_ForgotPassword.java:95)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:54)
at org.testng.internal.InvokeMethodRunnable.run(InvokeMethodRunnable.java:44)
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testForgotPassword -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="afterMethod" is-config="true" duration-ms="1" started-at="2020-05-18T11:36:42Z" finished-at="2020-05-18T11:36:42Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="FAIL" signature="testForgotPassword(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="testForgotPassword" duration-ms="2864" started-at="2020-05-18T11:36:42Z" data-provider="dataForgotPassword" finished-at="2020-05-18T11:36:45Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[negative case - invalid phone number format (not numerical)]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[not a number]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[testforgotpasswordbackend@gmail.com]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[400]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[Phone number not registered]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="7">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="8">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="9">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="10">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="11">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="12">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="13">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="14">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [400] but found [500]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [400] but found [500]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at org.testng.Assert.assertEquals(Assert.java:662)
at com.g2academy.base.Assertion.statusCode(Assertion.java:8)
at com.g2academy.testcases.loginmenu.TC_ForgotPassword.testForgotPassword(TC_ForgotPassword.java:95)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:54)
at org.testng.internal.InvokeMethodRunnable.run(InvokeMethodRunnable.java:44)
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testForgotPassword -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="afterMethod" is-config="true" duration-ms="0" started-at="2020-05-18T11:36:45Z" finished-at="2020-05-18T11:36:45Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="FAIL" signature="testForgotPassword(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="testForgotPassword" duration-ms="2895" started-at="2020-05-18T11:36:45Z" data-provider="dataForgotPassword" finished-at="2020-05-18T11:36:48Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[negative case - invalid phone number format (exceed 15 characters)]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[+628123456789101112222]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[testforgotpasswordbackend@gmail.com]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[400]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[Phone number not registered]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="7">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="8">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="9">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="10">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="11">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="12">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="13">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="14">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [400] but found [500]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [400] but found [500]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at org.testng.Assert.assertEquals(Assert.java:662)
at com.g2academy.base.Assertion.statusCode(Assertion.java:8)
at com.g2academy.testcases.loginmenu.TC_ForgotPassword.testForgotPassword(TC_ForgotPassword.java:95)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:54)
at org.testng.internal.InvokeMethodRunnable.run(InvokeMethodRunnable.java:44)
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testForgotPassword -->
        <test-method status="FAIL" signature="testForgotPassword(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="testForgotPassword" duration-ms="2632" started-at="2020-05-18T11:36:48Z" data-provider="dataForgotPassword" finished-at="2020-05-18T11:36:51Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[negative case - invalid phone number (outside Indonesia)]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[+601300808888]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[testforgotpasswordbackend@gmail.com]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[400]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[Phone number not registered]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="7">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="8">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="9">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="10">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="11">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="12">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="13">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="14">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [400] but found [500]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [400] but found [500]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at org.testng.Assert.assertEquals(Assert.java:662)
at com.g2academy.base.Assertion.statusCode(Assertion.java:8)
at com.g2academy.testcases.loginmenu.TC_ForgotPassword.testForgotPassword(TC_ForgotPassword.java:95)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:54)
at org.testng.internal.InvokeMethodRunnable.run(InvokeMethodRunnable.java:44)
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testForgotPassword -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="afterMethod" is-config="true" duration-ms="0" started-at="2020-05-18T11:36:48Z" finished-at="2020-05-18T11:36:48Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="afterMethod" is-config="true" duration-ms="0" started-at="2020-05-18T11:36:51Z" finished-at="2020-05-18T11:36:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="FAIL" signature="testForgotPassword(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="testForgotPassword" duration-ms="2811" started-at="2020-05-18T11:36:51Z" data-provider="dataForgotPassword" finished-at="2020-05-18T11:36:53Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[negative case - invalid phone number (unregistered number)]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[+6281234567890]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[testforgotpasswordbackend@gmail.com]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[400]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[Phone number not registered]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="7">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="8">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="9">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="10">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="11">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="12">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="13">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="14">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [400] but found [500]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [400] but found [500]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at org.testng.Assert.assertEquals(Assert.java:662)
at com.g2academy.base.Assertion.statusCode(Assertion.java:8)
at com.g2academy.testcases.loginmenu.TC_ForgotPassword.testForgotPassword(TC_ForgotPassword.java:95)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:54)
at org.testng.internal.InvokeMethodRunnable.run(InvokeMethodRunnable.java:44)
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testForgotPassword -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="afterMethod" is-config="true" duration-ms="1" started-at="2020-05-18T11:36:53Z" finished-at="2020-05-18T11:36:53Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="FAIL" signature="testForgotPassword(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="testForgotPassword" duration-ms="2972" started-at="2020-05-18T11:36:53Z" data-provider="dataForgotPassword" finished-at="2020-05-18T11:36:56Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[negative case - invalid phone number (unverified number)]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[+6285785161197]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[testforgotpasswordbackend@gmail.com]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[400]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[Phone number not registered]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="7">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="8">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="9">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="10">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="11">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="12">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="13">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="14">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [400] but found [500]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [400] but found [500]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at org.testng.Assert.assertEquals(Assert.java:662)
at com.g2academy.base.Assertion.statusCode(Assertion.java:8)
at com.g2academy.testcases.loginmenu.TC_ForgotPassword.testForgotPassword(TC_ForgotPassword.java:95)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:54)
at org.testng.internal.InvokeMethodRunnable.run(InvokeMethodRunnable.java:44)
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testForgotPassword -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="afterMethod" is-config="true" duration-ms="1" started-at="2020-05-18T11:36:56Z" finished-at="2020-05-18T11:36:56Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="PASS" signature="testForgotPassword(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="testForgotPassword" duration-ms="3073" started-at="2020-05-18T11:36:56Z" data-provider="dataForgotPassword" finished-at="2020-05-18T11:36:59Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[negative case - invalid email (no @)]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[+6281252930398]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[testforgotpassword.gmail.com]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[500]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[Internal Server Error]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="7">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="8">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="9">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="10">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="11">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="12">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="13">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="14">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testForgotPassword -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="afterMethod" is-config="true" duration-ms="1" started-at="2020-05-18T11:36:59Z" finished-at="2020-05-18T11:36:59Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="FAIL" signature="testForgotPassword(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="testForgotPassword" duration-ms="4335" started-at="2020-05-18T11:36:59Z" data-provider="dataForgotPassword" finished-at="2020-05-18T11:37:04Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[negative case - invalid otp code, status otp code]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[+6281252930398]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[testforgotpasswordbackend@gmail.com]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[OTP]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[12AX]]>
              </value>
            </param>
            <param index="7">
              <value>
                <![CDATA[false]]>
              </value>
            </param>
            <param index="8">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="9">
              <value>
                <![CDATA[500]]>
              </value>
            </param>
            <param index="10">
              <value>
                <![CDATA[Internal Server Error]]>
              </value>
            </param>
            <param index="11">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="12">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="13">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="14">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [200] but found [500]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [200] but found [500]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at org.testng.Assert.assertEquals(Assert.java:662)
at com.g2academy.base.Assertion.statusCode(Assertion.java:8)
at com.g2academy.testcases.loginmenu.TC_ForgotPassword.testForgotPassword(TC_ForgotPassword.java:95)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:54)
at org.testng.internal.InvokeMethodRunnable.run(InvokeMethodRunnable.java:44)
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testForgotPassword -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="afterMethod" is-config="true" duration-ms="0" started-at="2020-05-18T11:37:04Z" finished-at="2020-05-18T11:37:04Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="FAIL" signature="testForgotPassword(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="testForgotPassword" duration-ms="2854" started-at="2020-05-18T11:37:04Z" data-provider="dataForgotPassword" finished-at="2020-05-18T11:37:07Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[negative case - invalid otp code]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[+6281252930398]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[testforgotpasswordbackend@gmail.com]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[OTP]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[12AX]]>
              </value>
            </param>
            <param index="7">
              <value>
                <![CDATA[true]]>
              </value>
            </param>
            <param index="8">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="9">
              <value>
                <![CDATA[500]]>
              </value>
            </param>
            <param index="10">
              <value>
                <![CDATA[Internal Server Error]]>
              </value>
            </param>
            <param index="11">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="12">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="13">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="14">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [200] but found [500]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [200] but found [500]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at org.testng.Assert.assertEquals(Assert.java:662)
at com.g2academy.base.Assertion.statusCode(Assertion.java:8)
at com.g2academy.testcases.loginmenu.TC_ForgotPassword.testForgotPassword(TC_ForgotPassword.java:95)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:54)
at org.testng.internal.InvokeMethodRunnable.run(InvokeMethodRunnable.java:44)
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testForgotPassword -->
        <test-method status="FAIL" signature="testForgotPassword(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="testForgotPassword" duration-ms="2814" started-at="2020-05-18T11:37:07Z" data-provider="dataForgotPassword" finished-at="2020-05-18T11:37:09Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[negative case - invalid status otp code]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[+6281252930398]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[testforgotpasswordbackend@gmail.com]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[OTP]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[TRUE]]>
              </value>
            </param>
            <param index="7">
              <value>
                <![CDATA[false]]>
              </value>
            </param>
            <param index="8">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="9">
              <value>
                <![CDATA[500]]>
              </value>
            </param>
            <param index="10">
              <value>
                <![CDATA[Internal Server Error]]>
              </value>
            </param>
            <param index="11">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="12">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="13">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="14">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [200] but found [500]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [200] but found [500]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at org.testng.Assert.assertEquals(Assert.java:662)
at com.g2academy.base.Assertion.statusCode(Assertion.java:8)
at com.g2academy.testcases.loginmenu.TC_ForgotPassword.testForgotPassword(TC_ForgotPassword.java:95)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:54)
at org.testng.internal.InvokeMethodRunnable.run(InvokeMethodRunnable.java:44)
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testForgotPassword -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="afterMethod" is-config="true" duration-ms="0" started-at="2020-05-18T11:37:07Z" finished-at="2020-05-18T11:37:07Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="FAIL" signature="testForgotPassword(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="testForgotPassword" duration-ms="3044" started-at="2020-05-18T11:37:09Z" data-provider="dataForgotPassword" finished-at="2020-05-18T11:37:12Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[positive case - valid otp code, valid status otp code]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[+6281252930398]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[testforgotpasswordbackend@gmail.com]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[OTP]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[TRUE]]>
              </value>
            </param>
            <param index="7">
              <value>
                <![CDATA[true]]>
              </value>
            </param>
            <param index="8">
              <value>
                <![CDATA[TRUE]]>
              </value>
            </param>
            <param index="9">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="10">
              <value>
                <![CDATA[Your token has been reset]]>
              </value>
            </param>
            <param index="11">
              <value>
                <![CDATA[Password12345@@]]>
              </value>
            </param>
            <param index="12">
              <value>
                <![CDATA[Password12345@@]]>
              </value>
            </param>
            <param index="13">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="14">
              <value>
                <![CDATA[+6281252930398]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [200] but found [500]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [200] but found [500]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at org.testng.Assert.assertEquals(Assert.java:662)
at com.g2academy.base.Assertion.statusCode(Assertion.java:8)
at com.g2academy.testcases.loginmenu.TC_ForgotPassword.testForgotPassword(TC_ForgotPassword.java:95)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:54)
at org.testng.internal.InvokeMethodRunnable.run(InvokeMethodRunnable.java:44)
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testForgotPassword -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="afterMethod" is-config="true" duration-ms="0" started-at="2020-05-18T11:37:09Z" finished-at="2020-05-18T11:37:09Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="afterMethod" is-config="true" duration-ms="0" started-at="2020-05-18T11:37:12Z" finished-at="2020-05-18T11:37:12Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="FAIL" signature="testForgotPassword(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="testForgotPassword" duration-ms="2865" started-at="2020-05-18T11:37:12Z" data-provider="dataForgotPassword" finished-at="2020-05-18T11:37:15Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[negative case - invalid otp code (below 4 characters)]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[+6281252930398]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[testforgotpasswordbackend@gmail.com]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[OTP]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[253]]>
              </value>
            </param>
            <param index="7">
              <value>
                <![CDATA[true]]>
              </value>
            </param>
            <param index="8">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="9">
              <value>
                <![CDATA[500]]>
              </value>
            </param>
            <param index="10">
              <value>
                <![CDATA[Internal Server Error]]>
              </value>
            </param>
            <param index="11">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="12">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="13">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="14">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [200] but found [500]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [200] but found [500]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at org.testng.Assert.assertEquals(Assert.java:662)
at com.g2academy.base.Assertion.statusCode(Assertion.java:8)
at com.g2academy.testcases.loginmenu.TC_ForgotPassword.testForgotPassword(TC_ForgotPassword.java:95)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:54)
at org.testng.internal.InvokeMethodRunnable.run(InvokeMethodRunnable.java:44)
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testForgotPassword -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="afterMethod" is-config="true" duration-ms="0" started-at="2020-05-18T11:37:15Z" finished-at="2020-05-18T11:37:15Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="FAIL" signature="testForgotPassword(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="testForgotPassword" duration-ms="2661" started-at="2020-05-18T11:37:15Z" data-provider="dataForgotPassword" finished-at="2020-05-18T11:37:18Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[negative case - invalid otp code (exceed 4 characters)]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[+6281252930398]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[testforgotpasswordbackend@gmail.com]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[OTP]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[1697389]]>
              </value>
            </param>
            <param index="7">
              <value>
                <![CDATA[true]]>
              </value>
            </param>
            <param index="8">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="9">
              <value>
                <![CDATA[500]]>
              </value>
            </param>
            <param index="10">
              <value>
                <![CDATA[Internal Server Error]]>
              </value>
            </param>
            <param index="11">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="12">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="13">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="14">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [200] but found [500]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [200] but found [500]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at org.testng.Assert.assertEquals(Assert.java:662)
at com.g2academy.base.Assertion.statusCode(Assertion.java:8)
at com.g2academy.testcases.loginmenu.TC_ForgotPassword.testForgotPassword(TC_ForgotPassword.java:95)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:54)
at org.testng.internal.InvokeMethodRunnable.run(InvokeMethodRunnable.java:44)
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testForgotPassword -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="afterMethod" is-config="true" duration-ms="1" started-at="2020-05-18T11:37:18Z" finished-at="2020-05-18T11:37:18Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="FAIL" signature="testForgotPassword(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="testForgotPassword" duration-ms="3685" started-at="2020-05-18T11:37:18Z" data-provider="dataForgotPassword" finished-at="2020-05-18T11:37:22Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[negative case - invalid otp code (not numerical only)]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[+6281252930398]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[testforgotpasswordbackend@gmail.com]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[OTP]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[12AX]]>
              </value>
            </param>
            <param index="7">
              <value>
                <![CDATA[true]]>
              </value>
            </param>
            <param index="8">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="9">
              <value>
                <![CDATA[500]]>
              </value>
            </param>
            <param index="10">
              <value>
                <![CDATA[Internal Server Error]]>
              </value>
            </param>
            <param index="11">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="12">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="13">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="14">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [200] but found [500]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [200] but found [500]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at org.testng.Assert.assertEquals(Assert.java:662)
at com.g2academy.base.Assertion.statusCode(Assertion.java:8)
at com.g2academy.testcases.loginmenu.TC_ForgotPassword.testForgotPassword(TC_ForgotPassword.java:95)
at sun.reflect.GeneratedMethodAccessor131.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:54)
at org.testng.internal.InvokeMethodRunnable.run(InvokeMethodRunnable.java:44)
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testForgotPassword -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="afterMethod" is-config="true" duration-ms="1" started-at="2020-05-18T11:37:22Z" finished-at="2020-05-18T11:37:22Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="FAIL" signature="testForgotPassword(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="testForgotPassword" duration-ms="3199" started-at="2020-05-18T11:37:22Z" data-provider="dataForgotPassword" finished-at="2020-05-18T11:37:25Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[negative case - invalid status otp code (timeout)]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[+6281252930398]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[testforgotpasswordbackend@gmail.com]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[OTP]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[TRUE]]>
              </value>
            </param>
            <param index="7">
              <value>
                <![CDATA[false]]>
              </value>
            </param>
            <param index="8">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="9">
              <value>
                <![CDATA[500]]>
              </value>
            </param>
            <param index="10">
              <value>
                <![CDATA[Internal Server Error]]>
              </value>
            </param>
            <param index="11">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="12">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="13">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="14">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [200] but found [500]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [200] but found [500]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at org.testng.Assert.assertEquals(Assert.java:662)
at com.g2academy.base.Assertion.statusCode(Assertion.java:8)
at com.g2academy.testcases.loginmenu.TC_ForgotPassword.testForgotPassword(TC_ForgotPassword.java:95)
at sun.reflect.GeneratedMethodAccessor131.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:54)
at org.testng.internal.InvokeMethodRunnable.run(InvokeMethodRunnable.java:44)
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testForgotPassword -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="afterMethod" is-config="true" duration-ms="0" started-at="2020-05-18T11:37:25Z" finished-at="2020-05-18T11:37:25Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="FAIL" signature="testForgotPassword(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="testForgotPassword" duration-ms="2738" started-at="2020-05-18T11:37:25Z" data-provider="dataForgotPassword" finished-at="2020-05-18T11:37:28Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[negative case - invalid otp code]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[+6281252930398]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[testforgotpasswordbackend@gmail.com]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[OTP]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[1256]]>
              </value>
            </param>
            <param index="7">
              <value>
                <![CDATA[true]]>
              </value>
            </param>
            <param index="8">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="9">
              <value>
                <![CDATA[500]]>
              </value>
            </param>
            <param index="10">
              <value>
                <![CDATA[Internal Server Error]]>
              </value>
            </param>
            <param index="11">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="12">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="13">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="14">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [200] but found [500]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [200] but found [500]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at org.testng.Assert.assertEquals(Assert.java:662)
at com.g2academy.base.Assertion.statusCode(Assertion.java:8)
at com.g2academy.testcases.loginmenu.TC_ForgotPassword.testForgotPassword(TC_ForgotPassword.java:95)
at sun.reflect.GeneratedMethodAccessor131.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:54)
at org.testng.internal.InvokeMethodRunnable.run(InvokeMethodRunnable.java:44)
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testForgotPassword -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="afterMethod" is-config="true" duration-ms="0" started-at="2020-05-18T11:37:28Z" finished-at="2020-05-18T11:37:28Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="FAIL" signature="testForgotPassword(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="testForgotPassword" duration-ms="2659" started-at="2020-05-18T11:37:28Z" data-provider="dataForgotPassword" finished-at="2020-05-18T11:37:30Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[positif case - valid token]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[+6281252930398]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[testforgotpasswordbackend@gmail.com]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[TOKEN]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="7">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="8">
              <value>
                <![CDATA[TRUE]]>
              </value>
            </param>
            <param index="9">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="10">
              <value>
                <![CDATA[Your token has been reset]]>
              </value>
            </param>
            <param index="11">
              <value>
                <![CDATA[Password12345@@]]>
              </value>
            </param>
            <param index="12">
              <value>
                <![CDATA[Password12345@@]]>
              </value>
            </param>
            <param index="13">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="14">
              <value>
                <![CDATA[+6281252930398]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [200] but found [500]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [200] but found [500]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at org.testng.Assert.assertEquals(Assert.java:662)
at com.g2academy.base.Assertion.statusCode(Assertion.java:8)
at com.g2academy.testcases.loginmenu.TC_ForgotPassword.testForgotPassword(TC_ForgotPassword.java:95)
at sun.reflect.GeneratedMethodAccessor131.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:54)
at org.testng.internal.InvokeMethodRunnable.run(InvokeMethodRunnable.java:44)
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testForgotPassword -->
        <test-method status="FAIL" signature="testForgotPassword(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="testForgotPassword" duration-ms="2868" started-at="2020-05-18T11:37:30Z" data-provider="dataForgotPassword" finished-at="2020-05-18T11:37:33Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[negative case - invalid token]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[+6281252930398]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[testforgotpasswordbackend@gmail.com]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[TOKEN]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="7">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="8">
              <value>
                <![CDATA[token-salah]]>
              </value>
            </param>
            <param index="9">
              <value>
                <![CDATA[500]]>
              </value>
            </param>
            <param index="10">
              <value>
                <![CDATA[Internal Server Error]]>
              </value>
            </param>
            <param index="11">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="12">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="13">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="14">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [200] but found [500]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [200] but found [500]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at org.testng.Assert.assertEquals(Assert.java:662)
at com.g2academy.base.Assertion.statusCode(Assertion.java:8)
at com.g2academy.testcases.loginmenu.TC_ForgotPassword.testForgotPassword(TC_ForgotPassword.java:95)
at sun.reflect.GeneratedMethodAccessor131.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:54)
at org.testng.internal.InvokeMethodRunnable.run(InvokeMethodRunnable.java:44)
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testForgotPassword -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="afterMethod" is-config="true" duration-ms="0" started-at="2020-05-18T11:37:30Z" finished-at="2020-05-18T11:37:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="afterMethod" is-config="true" duration-ms="0" started-at="2020-05-18T11:37:33Z" finished-at="2020-05-18T11:37:33Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="FAIL" signature="testForgotPassword(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="testForgotPassword" duration-ms="2914" started-at="2020-05-18T11:37:33Z" data-provider="dataForgotPassword" finished-at="2020-05-18T11:37:36Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[negative case - invalid new password, invalid confirm new password]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[+6281252930398]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[testforgotpasswordbackend@gmail.com]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[OTP]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[TRUE]]>
              </value>
            </param>
            <param index="7">
              <value>
                <![CDATA[true]]>
              </value>
            </param>
            <param index="8">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="9">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="10">
              <value>
                <![CDATA[Your token has been reset]]>
              </value>
            </param>
            <param index="11">
              <value>
                <![CDATA[PasswordSalah@@]]>
              </value>
            </param>
            <param index="12">
              <value>
                <![CDATA[PasswordSalah@@]]>
              </value>
            </param>
            <param index="13">
              <value>
                <![CDATA[400]]>
              </value>
            </param>
            <param index="14">
              <value>
                <![CDATA[400]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [200] but found [500]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [200] but found [500]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at org.testng.Assert.assertEquals(Assert.java:662)
at com.g2academy.base.Assertion.statusCode(Assertion.java:8)
at com.g2academy.testcases.loginmenu.TC_ForgotPassword.testForgotPassword(TC_ForgotPassword.java:95)
at sun.reflect.GeneratedMethodAccessor131.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:54)
at org.testng.internal.InvokeMethodRunnable.run(InvokeMethodRunnable.java:44)
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testForgotPassword -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="afterMethod" is-config="true" duration-ms="0" started-at="2020-05-18T11:37:36Z" finished-at="2020-05-18T11:37:36Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="FAIL" signature="testForgotPassword(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="testForgotPassword" duration-ms="3014" started-at="2020-05-18T11:37:36Z" data-provider="dataForgotPassword" finished-at="2020-05-18T11:37:39Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[negative case - invalid new password, valid confirm new password]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[+6281252930398]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[testforgotpasswordbackend@gmail.com]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[OTP]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[TRUE]]>
              </value>
            </param>
            <param index="7">
              <value>
                <![CDATA[true]]>
              </value>
            </param>
            <param index="8">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="9">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="10">
              <value>
                <![CDATA[Your token has been reset]]>
              </value>
            </param>
            <param index="11">
              <value>
                <![CDATA[PasswordSalah@@]]>
              </value>
            </param>
            <param index="12">
              <value>
                <![CDATA[Password12345@@]]>
              </value>
            </param>
            <param index="13">
              <value>
                <![CDATA[400]]>
              </value>
            </param>
            <param index="14">
              <value>
                <![CDATA[400]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [200] but found [500]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [200] but found [500]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at org.testng.Assert.assertEquals(Assert.java:662)
at com.g2academy.base.Assertion.statusCode(Assertion.java:8)
at com.g2academy.testcases.loginmenu.TC_ForgotPassword.testForgotPassword(TC_ForgotPassword.java:95)
at sun.reflect.GeneratedMethodAccessor131.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:54)
at org.testng.internal.InvokeMethodRunnable.run(InvokeMethodRunnable.java:44)
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testForgotPassword -->
        <test-method status="FAIL" signature="testForgotPassword(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="testForgotPassword" duration-ms="2583" started-at="2020-05-18T11:37:39Z" data-provider="dataForgotPassword" finished-at="2020-05-18T11:37:42Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[negative case - valid new password, invalid confirm new password]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[+6281252930398]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[testforgotpasswordbackend@gmail.com]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[OTP]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[TRUE]]>
              </value>
            </param>
            <param index="7">
              <value>
                <![CDATA[true]]>
              </value>
            </param>
            <param index="8">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="9">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="10">
              <value>
                <![CDATA[Your token has been reset]]>
              </value>
            </param>
            <param index="11">
              <value>
                <![CDATA[Password12345@@]]>
              </value>
            </param>
            <param index="12">
              <value>
                <![CDATA[PasswordSalah@@]]>
              </value>
            </param>
            <param index="13">
              <value>
                <![CDATA[400]]>
              </value>
            </param>
            <param index="14">
              <value>
                <![CDATA[400]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [200] but found [500]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [200] but found [500]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at org.testng.Assert.assertEquals(Assert.java:662)
at com.g2academy.base.Assertion.statusCode(Assertion.java:8)
at com.g2academy.testcases.loginmenu.TC_ForgotPassword.testForgotPassword(TC_ForgotPassword.java:95)
at sun.reflect.GeneratedMethodAccessor131.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:54)
at org.testng.internal.InvokeMethodRunnable.run(InvokeMethodRunnable.java:44)
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testForgotPassword -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="afterMethod" is-config="true" duration-ms="0" started-at="2020-05-18T11:37:39Z" finished-at="2020-05-18T11:37:39Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="FAIL" signature="testForgotPassword(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="testForgotPassword" duration-ms="3157" started-at="2020-05-18T11:37:42Z" data-provider="dataForgotPassword" finished-at="2020-05-18T11:37:45Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[positive case - valid new password, valid confirm new password]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[+6281252930398]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[testforgotpasswordbackend@gmail.com]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[OTP]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="7">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="8">
              <value>
                <![CDATA[TRUE]]>
              </value>
            </param>
            <param index="9">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="10">
              <value>
                <![CDATA[Your token has been reset]]>
              </value>
            </param>
            <param index="11">
              <value>
                <![CDATA[Password12345@@]]>
              </value>
            </param>
            <param index="12">
              <value>
                <![CDATA[Password12345@@]]>
              </value>
            </param>
            <param index="13">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="14">
              <value>
                <![CDATA[+6281252930398]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [200] but found [500]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [200] but found [500]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at org.testng.Assert.assertEquals(Assert.java:662)
at com.g2academy.base.Assertion.statusCode(Assertion.java:8)
at com.g2academy.testcases.loginmenu.TC_ForgotPassword.testForgotPassword(TC_ForgotPassword.java:95)
at sun.reflect.GeneratedMethodAccessor131.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:54)
at org.testng.internal.InvokeMethodRunnable.run(InvokeMethodRunnable.java:44)
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testForgotPassword -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="afterMethod" is-config="true" duration-ms="0" started-at="2020-05-18T11:37:42Z" finished-at="2020-05-18T11:37:42Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="FAIL" signature="testForgotPassword(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="testForgotPassword" duration-ms="2914" started-at="2020-05-18T11:37:45Z" data-provider="dataForgotPassword" finished-at="2020-05-18T11:37:48Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[negative case - invalid new password (no lower case alphabetical character)]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[+6281252930398]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[testforgotpasswordbackend@gmail.com]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[OTP]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="7">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="8">
              <value>
                <![CDATA[TRUE]]>
              </value>
            </param>
            <param index="9">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="10">
              <value>
                <![CDATA[Your token has been reset]]>
              </value>
            </param>
            <param index="11">
              <value>
                <![CDATA[PASSWORD12345@@]]>
              </value>
            </param>
            <param index="12">
              <value>
                <![CDATA[Password12345@@]]>
              </value>
            </param>
            <param index="13">
              <value>
                <![CDATA[400]]>
              </value>
            </param>
            <param index="14">
              <value>
                <![CDATA[400]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [200] but found [500]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [200] but found [500]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at org.testng.Assert.assertEquals(Assert.java:662)
at com.g2academy.base.Assertion.statusCode(Assertion.java:8)
at com.g2academy.testcases.loginmenu.TC_ForgotPassword.testForgotPassword(TC_ForgotPassword.java:95)
at sun.reflect.GeneratedMethodAccessor131.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:54)
at org.testng.internal.InvokeMethodRunnable.run(InvokeMethodRunnable.java:44)
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testForgotPassword -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="afterMethod" is-config="true" duration-ms="0" started-at="2020-05-18T11:37:45Z" finished-at="2020-05-18T11:37:45Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="afterMethod" is-config="true" duration-ms="0" started-at="2020-05-18T11:37:48Z" finished-at="2020-05-18T11:37:48Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="FAIL" signature="testForgotPassword(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="testForgotPassword" duration-ms="3430" started-at="2020-05-18T11:37:48Z" data-provider="dataForgotPassword" finished-at="2020-05-18T11:37:51Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[negative case - invalid new password (no upper case alphabetical character)]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[+6281252930398]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[testforgotpasswordbackend@gmail.com]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[OTP]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="7">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="8">
              <value>
                <![CDATA[TRUE]]>
              </value>
            </param>
            <param index="9">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="10">
              <value>
                <![CDATA[Your token has been reset]]>
              </value>
            </param>
            <param index="11">
              <value>
                <![CDATA[password12345]]>
              </value>
            </param>
            <param index="12">
              <value>
                <![CDATA[Password12345@@]]>
              </value>
            </param>
            <param index="13">
              <value>
                <![CDATA[400]]>
              </value>
            </param>
            <param index="14">
              <value>
                <![CDATA[400]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [200] but found [500]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [200] but found [500]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at org.testng.Assert.assertEquals(Assert.java:662)
at com.g2academy.base.Assertion.statusCode(Assertion.java:8)
at com.g2academy.testcases.loginmenu.TC_ForgotPassword.testForgotPassword(TC_ForgotPassword.java:95)
at sun.reflect.GeneratedMethodAccessor131.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:54)
at org.testng.internal.InvokeMethodRunnable.run(InvokeMethodRunnable.java:44)
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testForgotPassword -->
        <test-method status="FAIL" signature="testForgotPassword(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="testForgotPassword" duration-ms="2254" started-at="2020-05-18T11:37:51Z" data-provider="dataForgotPassword" finished-at="2020-05-18T11:37:53Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[negative case - invalid new  password (no special character)]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[+6281252930398]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[testforgotpasswordbackend@gmail.com]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[OTP]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="7">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="8">
              <value>
                <![CDATA[TRUE]]>
              </value>
            </param>
            <param index="9">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="10">
              <value>
                <![CDATA[Your token has been reset]]>
              </value>
            </param>
            <param index="11">
              <value>
                <![CDATA[Password12345]]>
              </value>
            </param>
            <param index="12">
              <value>
                <![CDATA[Password12345@@]]>
              </value>
            </param>
            <param index="13">
              <value>
                <![CDATA[400]]>
              </value>
            </param>
            <param index="14">
              <value>
                <![CDATA[400]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [200] but found [500]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [200] but found [500]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at org.testng.Assert.assertEquals(Assert.java:662)
at com.g2academy.base.Assertion.statusCode(Assertion.java:8)
at com.g2academy.testcases.loginmenu.TC_ForgotPassword.testForgotPassword(TC_ForgotPassword.java:95)
at sun.reflect.GeneratedMethodAccessor131.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:54)
at org.testng.internal.InvokeMethodRunnable.run(InvokeMethodRunnable.java:44)
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testForgotPassword -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="afterMethod" is-config="true" duration-ms="0" started-at="2020-05-18T11:37:51Z" finished-at="2020-05-18T11:37:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="afterMethod" is-config="true" duration-ms="0" started-at="2020-05-18T11:37:53Z" finished-at="2020-05-18T11:37:53Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="FAIL" signature="testForgotPassword(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="testForgotPassword" duration-ms="2795" started-at="2020-05-18T11:37:53Z" data-provider="dataForgotPassword" finished-at="2020-05-18T11:37:56Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[negative case - invalid new password (no numerical character)]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[+6281252930398]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[testforgotpasswordbackend@gmail.com]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[OTP]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="7">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="8">
              <value>
                <![CDATA[TRUE]]>
              </value>
            </param>
            <param index="9">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="10">
              <value>
                <![CDATA[Your token has been reset]]>
              </value>
            </param>
            <param index="11">
              <value>
                <![CDATA[PasswordSalah@@]]>
              </value>
            </param>
            <param index="12">
              <value>
                <![CDATA[Password12345@@]]>
              </value>
            </param>
            <param index="13">
              <value>
                <![CDATA[400]]>
              </value>
            </param>
            <param index="14">
              <value>
                <![CDATA[400]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [200] but found [500]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [200] but found [500]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at org.testng.Assert.assertEquals(Assert.java:662)
at com.g2academy.base.Assertion.statusCode(Assertion.java:8)
at com.g2academy.testcases.loginmenu.TC_ForgotPassword.testForgotPassword(TC_ForgotPassword.java:95)
at sun.reflect.GeneratedMethodAccessor131.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:54)
at org.testng.internal.InvokeMethodRunnable.run(InvokeMethodRunnable.java:44)
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testForgotPassword -->
        <test-method status="FAIL" signature="testForgotPassword(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="testForgotPassword" duration-ms="2419" started-at="2020-05-18T11:37:56Z" data-provider="dataForgotPassword" finished-at="2020-05-18T11:37:59Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[negative case - invalid new password (below 8 characters)]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[+6281252930398]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[testforgotpasswordbackend@gmail.com]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[OTP]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="7">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="8">
              <value>
                <![CDATA[TRUE]]>
              </value>
            </param>
            <param index="9">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="10">
              <value>
                <![CDATA[Your token has been reset]]>
              </value>
            </param>
            <param index="11">
              <value>
                <![CDATA[Pwd123@]]>
              </value>
            </param>
            <param index="12">
              <value>
                <![CDATA[Password12345@@]]>
              </value>
            </param>
            <param index="13">
              <value>
                <![CDATA[400]]>
              </value>
            </param>
            <param index="14">
              <value>
                <![CDATA[400]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [200] but found [500]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [200] but found [500]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at org.testng.Assert.assertEquals(Assert.java:662)
at com.g2academy.base.Assertion.statusCode(Assertion.java:8)
at com.g2academy.testcases.loginmenu.TC_ForgotPassword.testForgotPassword(TC_ForgotPassword.java:95)
at sun.reflect.GeneratedMethodAccessor131.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:54)
at org.testng.internal.InvokeMethodRunnable.run(InvokeMethodRunnable.java:44)
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testForgotPassword -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="afterMethod" is-config="true" duration-ms="0" started-at="2020-05-18T11:37:56Z" finished-at="2020-05-18T11:37:56Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="afterMethod" is-config="true" duration-ms="0" started-at="2020-05-18T11:37:59Z" finished-at="2020-05-18T11:37:59Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="FAIL" signature="testForgotPassword(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="testForgotPassword" duration-ms="2512" started-at="2020-05-18T11:37:59Z" data-provider="dataForgotPassword" finished-at="2020-05-18T11:38:01Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[negative case - invalid new password (exceed 16 characters)]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[+6281252930398]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[testforgotpasswordbackend@gmail.com]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[OTP]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="7">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="8">
              <value>
                <![CDATA[TRUE]]>
              </value>
            </param>
            <param index="9">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="10">
              <value>
                <![CDATA[Your token has been reset]]>
              </value>
            </param>
            <param index="11">
              <value>
                <![CDATA[PasswordMelebihi16Karakter@@]]>
              </value>
            </param>
            <param index="12">
              <value>
                <![CDATA[Password12345@@]]>
              </value>
            </param>
            <param index="13">
              <value>
                <![CDATA[400]]>
              </value>
            </param>
            <param index="14">
              <value>
                <![CDATA[400]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [200] but found [500]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [200] but found [500]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at org.testng.Assert.assertEquals(Assert.java:662)
at com.g2academy.base.Assertion.statusCode(Assertion.java:8)
at com.g2academy.testcases.loginmenu.TC_ForgotPassword.testForgotPassword(TC_ForgotPassword.java:95)
at sun.reflect.GeneratedMethodAccessor131.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:54)
at org.testng.internal.InvokeMethodRunnable.run(InvokeMethodRunnable.java:44)
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testForgotPassword -->
        <test-method status="FAIL" signature="testForgotPassword(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="testForgotPassword" duration-ms="2515" started-at="2020-05-18T11:38:01Z" data-provider="dataForgotPassword" finished-at="2020-05-18T11:38:04Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[negative case - invalid confirm new password (no lower case alphabetical character)]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[+6281252930398]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[testforgotpasswordbackend@gmail.com]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[OTP]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="7">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="8">
              <value>
                <![CDATA[TRUE]]>
              </value>
            </param>
            <param index="9">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="10">
              <value>
                <![CDATA[Your token has been reset]]>
              </value>
            </param>
            <param index="11">
              <value>
                <![CDATA[Password12345@@]]>
              </value>
            </param>
            <param index="12">
              <value>
                <![CDATA[PASSWORD12345@@]]>
              </value>
            </param>
            <param index="13">
              <value>
                <![CDATA[400]]>
              </value>
            </param>
            <param index="14">
              <value>
                <![CDATA[400]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [200] but found [500]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [200] but found [500]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at org.testng.Assert.assertEquals(Assert.java:662)
at com.g2academy.base.Assertion.statusCode(Assertion.java:8)
at com.g2academy.testcases.loginmenu.TC_ForgotPassword.testForgotPassword(TC_ForgotPassword.java:95)
at sun.reflect.GeneratedMethodAccessor131.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:54)
at org.testng.internal.InvokeMethodRunnable.run(InvokeMethodRunnable.java:44)
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testForgotPassword -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="afterMethod" is-config="true" duration-ms="0" started-at="2020-05-18T11:38:01Z" finished-at="2020-05-18T11:38:01Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="afterMethod" is-config="true" duration-ms="1" started-at="2020-05-18T11:38:04Z" finished-at="2020-05-18T11:38:04Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="FAIL" signature="testForgotPassword(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="testForgotPassword" duration-ms="3062" started-at="2020-05-18T11:38:04Z" data-provider="dataForgotPassword" finished-at="2020-05-18T11:38:07Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[negative case - invalid confirm new password (no upper case alphabetical character)]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[+6281252930398]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[testforgotpasswordbackend@gmail.com]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[OTP]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="7">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="8">
              <value>
                <![CDATA[TRUE]]>
              </value>
            </param>
            <param index="9">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="10">
              <value>
                <![CDATA[Your token has been reset]]>
              </value>
            </param>
            <param index="11">
              <value>
                <![CDATA[Password12345@@]]>
              </value>
            </param>
            <param index="12">
              <value>
                <![CDATA[password12345]]>
              </value>
            </param>
            <param index="13">
              <value>
                <![CDATA[400]]>
              </value>
            </param>
            <param index="14">
              <value>
                <![CDATA[400]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [200] but found [500]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [200] but found [500]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at org.testng.Assert.assertEquals(Assert.java:662)
at com.g2academy.base.Assertion.statusCode(Assertion.java:8)
at com.g2academy.testcases.loginmenu.TC_ForgotPassword.testForgotPassword(TC_ForgotPassword.java:95)
at sun.reflect.GeneratedMethodAccessor131.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:54)
at org.testng.internal.InvokeMethodRunnable.run(InvokeMethodRunnable.java:44)
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testForgotPassword -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="afterMethod" is-config="true" duration-ms="0" started-at="2020-05-18T11:38:07Z" finished-at="2020-05-18T11:38:07Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="FAIL" signature="testForgotPassword(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="testForgotPassword" duration-ms="2485" started-at="2020-05-18T11:38:07Z" data-provider="dataForgotPassword" finished-at="2020-05-18T11:38:09Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[negative case - invalid confirm new password (no special character)]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[+6281252930398]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[testforgotpasswordbackend@gmail.com]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[OTP]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="7">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="8">
              <value>
                <![CDATA[TRUE]]>
              </value>
            </param>
            <param index="9">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="10">
              <value>
                <![CDATA[Your token has been reset]]>
              </value>
            </param>
            <param index="11">
              <value>
                <![CDATA[Password12345@@]]>
              </value>
            </param>
            <param index="12">
              <value>
                <![CDATA[Password12345]]>
              </value>
            </param>
            <param index="13">
              <value>
                <![CDATA[400]]>
              </value>
            </param>
            <param index="14">
              <value>
                <![CDATA[400]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [200] but found [500]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [200] but found [500]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at org.testng.Assert.assertEquals(Assert.java:662)
at com.g2academy.base.Assertion.statusCode(Assertion.java:8)
at com.g2academy.testcases.loginmenu.TC_ForgotPassword.testForgotPassword(TC_ForgotPassword.java:95)
at sun.reflect.GeneratedMethodAccessor131.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:54)
at org.testng.internal.InvokeMethodRunnable.run(InvokeMethodRunnable.java:44)
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testForgotPassword -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="afterMethod" is-config="true" duration-ms="1" started-at="2020-05-18T11:38:09Z" finished-at="2020-05-18T11:38:09Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="FAIL" signature="testForgotPassword(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="testForgotPassword" duration-ms="3001" started-at="2020-05-18T11:38:09Z" data-provider="dataForgotPassword" finished-at="2020-05-18T11:38:12Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[negative case - invalid confirm new password (no numerical character)]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[+6281252930398]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[testforgotpasswordbackend@gmail.com]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[OTP]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="7">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="8">
              <value>
                <![CDATA[TRUE]]>
              </value>
            </param>
            <param index="9">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="10">
              <value>
                <![CDATA[Your token has been reset]]>
              </value>
            </param>
            <param index="11">
              <value>
                <![CDATA[Password12345@@]]>
              </value>
            </param>
            <param index="12">
              <value>
                <![CDATA[PasswordSalah@@]]>
              </value>
            </param>
            <param index="13">
              <value>
                <![CDATA[400]]>
              </value>
            </param>
            <param index="14">
              <value>
                <![CDATA[400]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [200] but found [500]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [200] but found [500]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at org.testng.Assert.assertEquals(Assert.java:662)
at com.g2academy.base.Assertion.statusCode(Assertion.java:8)
at com.g2academy.testcases.loginmenu.TC_ForgotPassword.testForgotPassword(TC_ForgotPassword.java:95)
at sun.reflect.GeneratedMethodAccessor131.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:54)
at org.testng.internal.InvokeMethodRunnable.run(InvokeMethodRunnable.java:44)
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testForgotPassword -->
        <test-method status="FAIL" signature="testForgotPassword(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="testForgotPassword" duration-ms="2608" started-at="2020-05-18T11:38:12Z" data-provider="dataForgotPassword" finished-at="2020-05-18T11:38:15Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[negative case - invalid confirm new password (below 8 characters)]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[+6281252930398]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[testforgotpasswordbackend@gmail.com]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[OTP]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="7">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="8">
              <value>
                <![CDATA[TRUE]]>
              </value>
            </param>
            <param index="9">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="10">
              <value>
                <![CDATA[Your token has been reset]]>
              </value>
            </param>
            <param index="11">
              <value>
                <![CDATA[Password12345@@]]>
              </value>
            </param>
            <param index="12">
              <value>
                <![CDATA[Pwd123@]]>
              </value>
            </param>
            <param index="13">
              <value>
                <![CDATA[400]]>
              </value>
            </param>
            <param index="14">
              <value>
                <![CDATA[400]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [200] but found [500]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [200] but found [500]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at org.testng.Assert.assertEquals(Assert.java:662)
at com.g2academy.base.Assertion.statusCode(Assertion.java:8)
at com.g2academy.testcases.loginmenu.TC_ForgotPassword.testForgotPassword(TC_ForgotPassword.java:95)
at sun.reflect.GeneratedMethodAccessor131.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:54)
at org.testng.internal.InvokeMethodRunnable.run(InvokeMethodRunnable.java:44)
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testForgotPassword -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="afterMethod" is-config="true" duration-ms="0" started-at="2020-05-18T11:38:12Z" finished-at="2020-05-18T11:38:12Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="afterMethod" is-config="true" duration-ms="1" started-at="2020-05-18T11:38:15Z" finished-at="2020-05-18T11:38:15Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="FAIL" signature="testForgotPassword(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="testForgotPassword" duration-ms="2825" started-at="2020-05-18T11:38:15Z" data-provider="dataForgotPassword" finished-at="2020-05-18T11:38:18Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[negative case - invalid confirm new password (exceed 16 characters)]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[+6281252930398]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[testforgotpasswordbackend@gmail.com]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[OTP]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="7">
              <value>
                <![CDATA[-]]>
              </value>
            </param>
            <param index="8">
              <value>
                <![CDATA[TRUE]]>
              </value>
            </param>
            <param index="9">
              <value>
                <![CDATA[200]]>
              </value>
            </param>
            <param index="10">
              <value>
                <![CDATA[Your token has been reset]]>
              </value>
            </param>
            <param index="11">
              <value>
                <![CDATA[Password12345@@]]>
              </value>
            </param>
            <param index="12">
              <value>
                <![CDATA[PasswordMelebihi16Karakter@@]]>
              </value>
            </param>
            <param index="13">
              <value>
                <![CDATA[400]]>
              </value>
            </param>
            <param index="14">
              <value>
                <![CDATA[400]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [200] but found [500]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [200] but found [500]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at org.testng.Assert.assertEquals(Assert.java:662)
at com.g2academy.base.Assertion.statusCode(Assertion.java:8)
at com.g2academy.testcases.loginmenu.TC_ForgotPassword.testForgotPassword(TC_ForgotPassword.java:95)
at sun.reflect.GeneratedMethodAccessor131.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:54)
at org.testng.internal.InvokeMethodRunnable.run(InvokeMethodRunnable.java:44)
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testForgotPassword -->
        <test-method status="PASS" signature="afterClass()[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="afterClass" is-config="true" duration-ms="3023" started-at="2020-05-18T11:38:18Z" finished-at="2020-05-18T11:38:21Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="afterMethod" is-config="true" duration-ms="0" started-at="2020-05-18T11:38:18Z" finished-at="2020-05-18T11:38:18Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="PASS" signature="teardown()[pri:0, instance:com.g2academy.testcases.loginmenu.TC_ForgotPassword@22f71333]" name="teardown" is-config="true" duration-ms="0" started-at="2020-05-18T11:38:21Z" finished-at="2020-05-18T11:38:21Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- com.g2academy.testcases.loginmenu.TC_ForgotPassword -->
    </test> <!-- QuotaAppTest -->
  </suite> <!-- Default Suite -->
</testng-results>
